L1Trigger/L1TCaloLayer1/src/L1TCaloLayer1FetchLUTs.cc

```diff
@@ -282,9 +300,27 @@ bool L1TCaloLayer1FetchLUTs(const edm::EventSetup& iSetup,
           if(useCalib) calibratedHFInput *= hfSF.at(phiBin*hfScalePhiBins.size()*12+etBin*12+etaBin);
           if(useLSB) calibratedHFInput /= caloLSB;
 
-          value = calibratedHFInput;
-          if(value > 0xFF) {
-            value = 0xFF;
+          if ( fwVersion > 2 ) {
+            uint32_t absCaloEta = abs(caloEta);
+            if(absCaloEta > 29 && absCaloEta < 40) {
+              // Divide by two (since two duplicate towers are sent)
+              calibratedHFInput /= 2.;
```

Comment:
Is this a style thing?
`cat > div.cpp`
```cpp
void div(double& x) {
  x /= 2.;
}
```
`gcc -S div.cpp`
```asm
...
	divsd	%xmm0, %xmm1
...
```
`gcc -O2 -S div.cpp`
```asm
...
	mulsd	LCPI0_0(%rip), %xmm0
...
```